specification/operationalinsights/resource-manager/Microsoft.OperationalInsights/stable/2020-10-01/Workspaces.json

```diff
@@ -424,12 +424,6 @@
           "format": "int32",
           "description": "The capacity reservation level for this workspace, when CapacityReservation sku is selected."
         },
-        "maxCapacityReservationLevel": {
-          "type": "integer",
-          "format": "int32",
-          "description": "The maximum capacity reservation level available for this workspace, when CapacityReservation sku is selected.",
-          "readOnly": true
-        },
         "lastSkuUpdate": {
```

Comment:
Can I know what is the reason to remove it? it is a breaking change which need to breaking change review board to approval

Reply:
> Hi @akning-ms,
> 
> I see that some validations are failing, can you pls help me understand the error?
> on Avocado - it complain on INCONSISTENT_API_VERSION although I didnt detect any inconcinnities
> Regarding MaxCapacity, it a read only property that was removed.
> 
> I need to advance the PR asap, I'd appreciate your help with it

It is due to the version in info section is wrong as below:
  "info": {
    "title": "Azure Log Analytics",
    "description": "Azure Log Analytics API reference",
    "version": "**2020-10-01**"
  },

Reply:
1. Sure, The reason for the readonly  maxCapacityRemoval is a change in pricing tier for log analytics workspace & cluster.  (the propery was already removed for ws ->https://github.com/Azure/azure-rest-api-specs/pull/13812 ).
Today with the new api-version we are introducing new enum for  simplification of the billing process and avoid over pricing on overage.

2. Thanks, fixed

Reply:
Breaking change template (Approved) -> https://msazure.visualstudio.com/One/_workitems/edit/10050734

Reply:
Looks the approval is not about "maxCapacity"